test/upload_validation_test.ts

```diff
@@ -1689,10 +1689,13 @@ describe('Pack metadata Validation', () => {
           type: AuthenticationType.CodaApiHeaderBearerToken,
           deferConnectionSetup: true,
           shouldAutoAuthSetup: true,
+          networkDomain: 'other.domain',
         },
         networkDomains: ['coda.io', 'other.domain'],
       });
       const err = await validateJsonAndAssertFails(metadata);
+      // TODO(dweitzman): Get rid of this error when credential pinning is launched
```

Comment:
Also added a TODO to this unit test. We can theoretically relax the CodaApiHeaderBearerToken validation to only look at the auth `networkDomain` once the enforcement of auth `networkDomain` is live